### PR TITLE
adds modal with instructions

### PR DESCRIPTION
**Pull request by Joperes-ym**
button to open instructions
event listener to display modal
event listener to close modal

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <link rel="stylesheet" type="text/css"  href="style.css" >
+  <script src="index.js" defer></script>
+</head>
+<body>
+<!-- Trigger/Open The Modal -->
+<button id="myBtn">Instructions</button>
```
Comment (HadertS) [inline index.html]: rename myBtn to something more semantic ie instructionBtn

File: style.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+/* Modal Content */
+#modal-content {
+    background-color: #fefefe;
+    margin: auto;
+    padding: 20px;
+    border: 1px solid #888;
+    width: 30%;
+    height: 50%
+
```
Comment (Yas-Shen) [inline style.css]: Needs a } here.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+<!-- The Modal -->
+<div id="modal">
+
+    <!-- Modal content -->
+    <div id="modal-content">
+        <span id="close">&times;</span>
+        <h2>INSTRUCTIONS</h2>
+        <p>You have 30 seconds to guess as many English counties as possible. You can guess as many times as you are able too.<br>Type your guess into the text box, if it is correct it will disappear, if it doesn't you've got it wrong - keep trying. Don't worry about capitalisation! <br>Correct guesses will be displayed in a list so you know which ones you've already got. <br>Click the "Start Game" button to start playing</p>
```
Comment (HadertS) [inline index.html]: \<br> tags should be removed and handled in the CSS

File: index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+
+// Get the modal
+let modal = document.getElementById("modal");
```
Comment (HadertS) [inline index.js]: rename modal to something more semantic ie InstructionModal.
Use querySelector instead of .getElementByID

File: index.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    <title>Title</title>
+    <link rel="stylesheet" type="text/css"  href="style.css" >
+  <script src="index.js" defer></script>
+</head>
+<body>
+<!-- Trigger/Open The Modal -->
+<button id="myBtn">Instructions</button>
+
+<!-- The Modal -->
+<div id="modal">
```
Comment (HadertS) [inline index.html]: as we're going to be using another modal later I would give this a more semantic id, ie instructionModal

File: index.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+// Get the button that opens the modal
+let btn = document.getElementById("myBtn");
```
Comment (HadertS) [inline index.js]: same as above regarding semantics and .getElementByID

File: index.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+// Get the <span> element that closes the modal
+let span = document.getElementById("close");
```
Comment (HadertS) [inline index.js]: same as above regarding semantics and .getElementByID

File: style.css [hunk 8]
```diff
@@ -0,0 +1,37 @@
+#modal {
+    display: none; /* Hidden by default */
+    position: fixed; /* Stay in place */
+    z-index: 1; /* Sit on top */
+    padding-top: 100px; /* Location of the box */
```
Comment (HadertS) [inline style.css]: consider using rem and 62.5% css trick instead of pixel value

File: style.css [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+    background-color: rgb(0,0,0); /* Fallback color */
+    background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+}
+
+/* Modal Content */
+#modal-content {
+    background-color: #fefefe;
+    margin: auto;
+    padding: 20px;
+    border: 1px solid #888;
```
Comment (HadertS) [inline style.css]: ditto regarding pixel value

File: style.css [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+    overflow: auto; /* Enable scroll if needed */
+    background-color: rgb(0,0,0); /* Fallback color */
+    background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+}
+
+/* Modal Content */
+#modal-content {
+    background-color: #fefefe;
+    margin: auto;
+    padding: 20px;
```
Comment (HadertS) [inline style.css]: ditto regarding pixel value

File: style.css (hunk 2 above)
Comment (HadertS) [reply style.css]: and a semicolon

Comment (HadertS) [reply style.css]: This breaks all the css after these lines.

File: style.css [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+    border: 1px solid #888;
+    width: 30%;
+    height: 50%
+
+/* The Close Button */
+#close {
+    color: #aaaaaa;
+    float: right;
+    font-size: 28px;
```
Comment (HadertS) [inline style.css]: ditto regarding pixel value

**Review by HadertS — CHANGES_REQUESTED** [review]
some changes to be made, let me know your thoughts regarding the pixel value vs rem as this is a preference thing for me.

**Review by HadertS — APPROVED** [review]

**Pull request merged**